### ISSUE TITLE
GBE+ Accesory Paritiy

### ISSUE THREAD

**Issue by ReNardier** [mgba-emu/mgba#2690]
if it ain't rude for me to ask, but i have been re-reading the edge of emulation articles and i have been thinking, that not many of them end up in mgba, the megaman carts was a big one that did, but i wonder wen is the turn of the others, feel free to answer with just the word "eventually" if it feels apropos
thanks for your time and for what you do

**Comment by endrift** [comment]
Lemme pull the list from Dan Docs and give you my plans:

- DMG and GBC
  - [Barcode Boy](https://shonumi.github.io/dandocs.html#bcb): No plans at the moment
  - [Barcode Taisen Bardigun Scanner](https://shonumi.github.io/dandocs.html#btb): No plans at the moment
  - [DMG-07 4-Player Adapter](https://shonumi.github.io/dandocs.html#dmg07): Potentially, once multplayer is further along
  - [Zok Zok Heroes Full Changer](https://shonumi.github.io/dandocs.html#zzh): No plans at the moment
  - [GBC Infrared Communication](https://shonumi.github.io/dandocs.html#ir): Planned before or in 1.0
  - [Gyogun Tanchiki: Pocket Sonar](https://shonumi.github.io/dandocs.html#ps): No plans at the moment
  - [Power Antenna + Bug Sensor](https://shonumi.github.io/dandocs.html#pabs): I actually have one of these and intend to implement it on the sooner side
  - [Turbo File GB](https://shonumi.github.io/dandocs.html#tbgb): No plans at the moment
  - [Sewing Machines](https://shonumi.github.io/dandocs.html#sew): Probably not going to implement this
- GBA
  - [GBA Screen Stretch](https://shonumi.github.io/dandocs.html#gss): Probably not going to implement this
  - [Soul Doll Adapter](https://shonumi.github.io/dandocs.html#sda): No plans at the moment
  - [Battle Chip Gate](https://shonumi.github.io/dandocs.html#bcg): Already done
  - [Multi Plust On System](https://shonumi.github.io/dandocs.html#mpos): No plans at the moment
  - [Turbo File Advance](https://shonumi.github.io/dandocs.html#tbfa): No plans at the moment
  - [AGB-006](https://shonumi.github.io/dandocs.html#agb006): I don't intend to implement this since it doesn't do anything without one of the Zoids toys
  - [Virtureal Racing System](https://shonumi.github.io/dandocs.html#vrs): No plans at the moment
  - [Magical Watch](https://shonumi.github.io/dandocs.html#maw): No plans at the moment
  - [Joy Carry Cartridge](https://shonumi.github.io/dandocs.html#jcc): I intend to look at this one on the sooner side, especially since I have one
  - [Advance Movie Adapter](https://shonumi.github.io/dandocs.html#ama): No plans at the moment
 
Note that "no plans" doesn't mean "it won't happen", it just means it's not a priority and haven't looked into it much.

There are also a few things that I know are being worked on that I have various plans about, including the Play-Yan and the Campho Advance.

**Comment by ReNardier** [comment]
"sewing and 006"
really?, not even a rendition the cheeky displays that shonumi implemented?, comprehensible
https://www.youtube.com/watch?v=NIZGflArRHw
https://shonumi.github.io/articles/zoids_8.gif
"stretch"
pity but comprehensible, i suppose that gba mode isn't really that used by people instead of the others to play gb/c games, i guess it's to much trouble for little to no reward
oh, and don't forget about the mobile adapter gb, apparently 16 gba games could us it as well
https://shonumi.github.io/dandocs.html#magb
all in all, good to know

**Comment by endrift** [comment]
I didn't forget about the Mobile Adapter. It's a whole separate thing that people are working on; I know there's some work into getting it working in mGBA too.

**Comment by HilbertIdeals5** [comment]
Does that mean that a future release of mGBA will support the Mobile Adapter?

**Comment by endrift** [comment]
Yes, @arves100 has been doing some stuff on this, though I don't know what the current status is.

**Comment by HilbertIdeals5** [comment]
From [this YouTube video](https://www.youtube.com/watch?v=oqsJQzywR50), it seems like something worked out. But arves100's repository (which the video links to) appears to be a read-only public archive as of Christmas last year.

Will any of that functionality be incorporated into this main/official repo? Or will it be exclusive to arves100's fork for the foreseeable future?

**Comment by endrift** [comment]
I don't know what the current status is, sorry. I do want to integrate it into mainline mGBA at some point, but I don't know when.

**Comment by arves100** [comment]
Hello, as far as I remember, there's currently new people working on the mobile adapter support (https://github.com/Wit-MKW/mgba) that's why my repo is archived
